pandapower/create.py

```diff
@@ -1981,23 +1987,21 @@ def create_lines(net, from_buses, to_buses, length_km, std_type, name=None, inde
     # add std type data
     if isinstance(std_type, str):
         lineparam = load_std_type(net, std_type, "line")
-        entries["r_ohm_per_km"] = lineparam["r_ohm_per_km"]
-        entries["x_ohm_per_km"] = lineparam["x_ohm_per_km"]
-        entries["c_nf_per_km"] = lineparam["c_nf_per_km"]
-        entries["max_i_ka"] = lineparam["max_i_ka"]
-        entries["g_us_per_km"] = lineparam["g_us_per_km"] if "g_us_per_km" in lineparam else 0.
-        if "type" in lineparam:
-            entries["type"] = lineparam["type"]
+        lineparam["g_us_per_km"] = lineparam["g_us_per_km"] if "g_us_per_km" in lineparam else 0
```

Comment:
I think we should not having this kind of logic here. If some types in pp std_type library are missing g_us_per_km, we should change it there (even if to 0).

Reply:
The user should be responsible for providing the relevant parameters in std_type, I think